mkt/files/tests/test_helpers.py

```diff
@@ -272,8 +272,9 @@ def change(self, file, text, filename='install.js'):
         open(path, 'w').write(data)
 
 
-class TestSafeUnzipFile(amo.tests.TestCase, amo.tests.AMOPaths):
+class TestSafeUnzipFile(mkt.site.tests.TestCase, mkt.site.tests.MktPaths):
 
+    ##(ej)## is any of this actually used? Its all testing for addons files
```

Comment:
Yes, we should. It should be updated to use a packaged app instead, but it's definitely useful.